clients/js/test/plugins/collection/masterEdition.test.ts

```diff
@@ -0,0 +1,155 @@
+import test from 'ava';
+import { none, some } from '@metaplex-foundation/umi';
+import {
+  pluginAuthorityPair,
+  updatePluginAuthority,
+  createPlugin,
+  addPluginV1,
+} from '../../../src';
+import {
+  DEFAULT_COLLECTION,
+  assertCollection,
+  createAsset,
+  createCollection,
+  createUmi,
+} from '../../_setup';
+
+test('it can add masterEdition to collection', async (t) => {
+  const umi = await createUmi();
+
+  const collection = await createCollection(umi, {
+    plugins: [
+      pluginAuthorityPair({
+        type: 'MasterEdition',
+        data: {
+          maxSupply: 100,
+          name: 'name',
+          uri: 'uri',
+        },
+        authority: updatePluginAuthority(),
+      }),
+    ],
+  });
+
+  await assertCollection(t, umi, {
+    ...DEFAULT_COLLECTION,
+    collection: collection.publicKey,
+    updateAuthority: umi.identity.publicKey,
+    masterEdition: {
+      authority: {
+        type: 'UpdateAuthority',
+      },
+      maxSupply: 100,
+      name: some('name'),
+      uri: some('uri'),
+    },
+  });
+});
+
+test('it can create collection with masterEdition', async (t) => {
+  const umi = await createUmi();
+
+  const collection = await createCollection(umi, {
+    plugins: [
+      pluginAuthorityPair({
+        type: 'MasterEdition',
+        data: {
+          maxSupply: 100,
+          name: 'name',
+          uri: 'uri',
+        },
+        authority: updatePluginAuthority(),
+      }),
+    ],
+  });
+
+  await assertCollection(t, umi, {
+    ...DEFAULT_COLLECTION,
+    collection: collection.publicKey,
+    updateAuthority: umi.identity.publicKey,
+    masterEdition: {
+      authority: {
+        type: 'UpdateAuthority',
+      },
+      maxSupply: 100,
+      name: some('name'),
+      uri: some('uri'),
+    },
+  });
+});
```

Comment:
Aren't those two tests exactly the same?
Were you going for add and update tests here?

Reply:
good catch, will update